### DISCUSSION TITLE
MobArena - Mobs

### DISCUSSION THREAD

**Discussion by Batatoones** [leogianfagna/IssueTracker#930] [Ideia]
### Descrição da tarefa

Os creepers charges da mobarena estão honestamente imoral de sobreviver. Qualquer classe acaba morrendo com aquele mob além da quantidade absurda que é spawnada

**Comment by Batatoones** [comment]
@Micosedeunha

**Comment by leogianfagna** [comment]
@Batatoones o arqueiro teoricamente não mata facil?

**Comment by Batatoones** [comment]
@leogianfagna não da pra sobreviver sendo que o range de explosão é grande além de spawnar pela arena toda.

**Comment by Micosedeunha** [comment]
Ideias:

- **Mago:** vara que solta a bola de neve (ai a bola de neve explode os bixo, ou taca fogo soh)
- **Tanque:** comecar com espada de pedra e dps upar para ferro || habilidade especial: _supertranquilo_ (recebe efeito da tartaruga)
- **Cavaleiro:** espada vai ficando encantada e melhor || habilidade especial: _espada divina_ defere um golpe em area (da pra fazer?)
- **Bárbaro:** machado vai ficando encantado e melhor || habilidade especial: _raiva ou fúria_ (recebe forca 3 e velocidade 2 por um tempo)
- **Arqueiro:** arco vai ficando encantado e melhor || habilidade especial: _reposicionar_ fica invisível para se reposicionar (talvez mais alguma coisa tbm)
- **Suporte:** recebe um cajado de cura (pra eu n ter q ficar adicionando wave de suplemento pra esse caraio) e um cajado de assistência || habilidade especial: _anjo da guarda_ deixa todos os jogadores com absorção e resistência (cd alto tbm pq ai se tiver um monte vai todo mundo pegar essa coisa e ninguém vai morrer mais)
- **NPC**: Easter egg maximo quem sobreviver com isso ate a wave 40 vai ganhar o CONVERSA (stick q da ik) (detalhe ele continua pelado e vuneravel)

**Comment by leogianfagna** [comment]
Boas ideias. Programação falando, da pra fazer de duas maneiras

- ele só explode as coisas se tiver encantado com sharpness 2 ou mais
- entrega uma espada/item completamente novo na wave

A mobarena consegue entregar itens com nbt tag ne?
